server/routes/planOverview/PlanOverviewController.ts

```diff
@@ -49,6 +49,7 @@ export default class PlanOverviewController {
       return res.render(page, {
         locale: locale.en,
         data: {
+          planAgreementStatus: this.plan.agreementStatus, // required by layout.njk
```

Comment:
As mentioned on Slack, considering this is now being required at the base level for all templates, it's likely worth us exploring a better means to provide this data to views rather than requiring it in every controller. One for a follow-up ticket.